pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -524,16 +536,13 @@ func Run() {
 	updateInprogress := isBaseOsCurrentPartitionStateInProgress(&zedagentCtx)
 	log.Infof("Current partition inProgress state is %v\n", updateInprogress)
 	log.Infof("Waiting until we have some uplinks with usable addresses\n")
-	for !DNSctx.DNSinitialized ||
-		!zedagentCtx.assignableAdapters.Initialized {
+	for !DNSctx.DNSinitialized {
 
-		// XXX AA will not initialize if bad json file
 		// XXX in that case send fails with no management ports
```

Comment:
Please remove the remaining 3 lines of the comment; you just removed the first line.

Reply:
Done